zebedee-cms/src/main/java/com/github/onsdigital/zebedee/permissions/service/JWTPermissionsServiceImpl.java

```diff
@@ -0,0 +1,282 @@
+     * @throws JWTVerificationException
+     * @deprecated with dp-identity-api
+     */
+    @Override
+    public void removeEditor(String email, Session session) throws JWTVerificationException {
+        throw new JWTVerificationException(JWTPERMISSIONSSERVICE_ERROR);
+    }
+
+    @Override
+    public boolean canView(Session session, CollectionDescription collectionDescription) throws JWTVerificationException {
+        throw new JWTVerificationException(JWTPERMISSIONSSERVICE_ERROR);
+    }
+
+    @Override
+    public boolean canView(User user, CollectionDescription collectionDescription) throws JWTVerificationException {
+        throw new JWTVerificationException(JWTPERMISSIONSSERVICE_ERROR);
+    }
+
+    @Override
+    public boolean canView(String email, CollectionDescription collectionDescription) throws JWTVerificationException {
+        throw new JWTVerificationException(JWTPERMISSIONSSERVICE_ERROR);
+    }
+
+    /**
+     * @param collectionDescription the {@link CollectionDescription} of the {@link Collection} in question.
+     * @param team                  the {@link Team} to permit view permission to.
+     * @param session               the {@link Session} of the user granting the permission.
+     * @throws JWTVerificationException
+     * @deprecated with dp-identity-api
+     */
+    @Override
+    public void addViewerTeam(CollectionDescription collectionDescription, Team team, Session session) throws JWTVerificationException {
+        throw new JWTVerificationException(JWTPERMISSIONSSERVICE_ERROR);
+    }
+
+    @Override
+    public Set<Integer> listViewerTeams(CollectionDescription collectionDescription, Session session) throws JWTVerificationException {
+        throw new JWTVerificationException(JWTPERMISSIONSSERVICE_ERROR);
+    }
+
+    /**
+     * @param collectionDescription the {@link CollectionDescription} of the {@link Collection} to remove the team.
+     * @param team                  the {@link Team} to remove.
+     * @param session               the {@link Session} of the user revoking view permission.
+     * @throws JWTVerificationException
+     * @deprecated with dp-identity-api
+     */
+    @Override
+    public void removeViewerTeam(CollectionDescription collectionDescription, Team team, Session session) throws JWTVerificationException {
+        throw new JWTVerificationException(JWTPERMISSIONSSERVICE_ERROR);
+    }
+
+    @Override
+    public PermissionDefinition userPermissions(String email, Session session) throws JWTVerificationException {
+        throw new JWTVerificationException(JWTPERMISSIONSSERVICE_ERROR);
+    }
+
+    @Override
+    public Set<String> listCollectionsAccessibleByTeam(Team t) throws JWTVerificationException {
+        throw new JWTVerificationException(JWTPERMISSIONSSERVICE_ERROR);
+    }
+
+
+    /**
+     * @param session
+     * @param permission
+     * @return
+     */
+    public boolean hasPermission(Session session, String permission) {
```

Comment:
The name of this method is a little misleading since it is actually checking if a session is in a group. Also, this method should probably be private.
Perhaps something like:

```suggestion
    private boolean isGroupMember(Session session, String group) {
```